extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -35,3 +35,87 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
   opacity: 0.55 !important;
   transition: opacity 0.2s cubic-bezier(0, 0, 0.2, 1) !important;
 }
+
+/** Make the dialog wrapper from amp-subscriptions round corner. */
+amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
+  border-radius: 16px 16px 0px 0px;
+}
+
+.i-amphtml-story-subscriptions-banner {
+  position: relative !important;
+  width: 100% !important;
+  background: #EBDEC9;
+  border-radius: 16px 16px 0px 0px;
+  font-family: Poppins;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 24px;
+  line-height: 24px;
```

Comment:
I actually get these from the UX mocks. They are always one-line so that's why it looks quite normal in the mocks. For now, let me just remove those line-height properties. It would be updated with new mocks in future PRs anyway.

The ideal UX here would be limiting it to just one line. What would be the best approach to limit the string length? I am thinking checking the number of characters in buildCallback() and stop constructing the extension if it exceeds the pre-defined number, which could be based on the font size. What do you think?